src/App.tsx

```diff
@@ -1,31 +1,56 @@
+import { useState } from 'react';
+import { Link, Route, Routes } from 'react-router-dom';
 import './App.scss';
-
-/*
-import { RouteComponentProps } from 'react-router-dom';
-
-type TabsPageProps = React.FC<RouteComponentProps<{ tabId: string }>>;
-const TabsPage: TabsPageProps = ({ match }) => {...};
-
-or
-
-import { useParams } from 'react-router-dom';
-
-const TabsPage = () => {
-  const { tabId } = useParams<{ tabId: string }>();
-  ...
+import TabsPage from './TabsPage';
+
+const App: React.FC = () => {
+  const [linkIsActive, setLinkIsActive] = useState('home');
+
+  return (
+    <div className="App">
+      <nav>
+        <Link
+          onClick={() => {
+            setLinkIsActive('home');
+          }}
+          className={linkIsActive === 'home' ? 'link link--active' : 'link'}
+          to="/"
+        >
+          Home page
+        </Link>
```

Comment:
use NabLink if you want to see an active link and remove state `linkIsActive` with NavLink you don't need this